checker/return_error_last_checker.go

```diff
@@ -18,6 +18,26 @@ func NewReturnErrorLastChecker(configData interface{}) NodeChecker {
 	return &ReturnErrorLastChecker{}
 }
 
+// Title implements the NodeChecker interface.
+func (c *ReturnErrorLastChecker) Title() string {
+	return "Error Return Last"
```

Comment:
It seems to me that `Return Error Last` is clearer.

Reply:
Done.